src/content/operations/monitoring/_index.en.md

```diff
@@ -20,6 +24,26 @@ The following metrics are exposed currently:
   * `remote_cluster`: the remote cluster name
   * `remote_hostname`: the remote hostname
   * `status`: the connection status (“connecting”, “connected”, or “error”)
+  
+### Submariner
```

Comment:
@maayanf24 @skitt Technically, we expose the metrics for `submariner operator` and `submariner` separately as shown here. But should the user care about this? Should we just provide a list of all the available metrics or group them somehow by functionality?

Reply:
Right, the user shouldn’t care, all the metrics appear in the same Prometheus UI.

Reply:
I'll add them all in one list for now, later when we have some more we can think of grouping related metrics